add-on/chrome/bootstrap.js

```diff
@@ -126,49 +125,15 @@ var WindowListener = {
        *
        * @param {event}  event   The event opening the panel, used to anchor
        *                         the panel to the button which triggers it.
-       * @param {String} [tabId] Identifier of the tab to select when the panel is
-       *                         opened. Example: 'rooms', 'contacts', etc.
        * @return {Promise}
        */
-      openCallPanel: function(event, tabId = null) {
+      openCallPanel: function(event) {
```

Comment:
Do you mind renaming this whilst you're touching it, e.g. openPanel or something? It feels better that way as its not just about calls...